shells/katello_p4.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# *******
+# This is a script to get katello installed with puppet 4. It is only meant to
+# be a POC and should not be used in production scenarios.
+#
+# This script is temporary and should go away and be replaced with something
+# better.
+# *******
+
+yum -y localinstall http://yum.theforeman.org/releases/1.12/el7/x86_64/foreman-release.rpm
+yum -y localinstall http://fedorapeople.org/groups/katello/releases/yum/3.1/katello/el7/x86_64/katello-repos-latest.rpm
+yum -y localinstall https://yum.puppetlabs.com/puppetlabs-release-pc1-el-7.noarch.rpm # puppet 4!!!
+yum -y localinstall http://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+yum -y install foreman-release-scl
+yum -y install katello
+yum -y install puppet-agent #this upgrades puppet
+
+yum install -y haveged; service haveged start # saves some time on tomcat startup, worth it:)
+
+# XXX: please fix this second
+setenforce 0  # uggggh! stopdisablingselinux.com :-P
+
+# we need "puppet help strings" to work so kafo_parsers picks up the parser
+/opt/puppetlabs/bin/puppet module install puppetlabs-strings
+/opt/puppetlabs/puppet/bin/gem install yard
+
+# at this point, "foreman-installer -v --noop --scenario katello" will run (not clean, but runs)
+
+service mongod start # to get around error related to master selection in mongo puppet module
```

Comment:
I was trying to troubleshoot this, so I removed this line - but I still get a clean install on a fresh box.

```
[ INFO 2016-08-02 11:19:00 verbose] Executing hooks in group post
  Success!
  * Katello is running at https://centos7-katello-p4.example.com
      Initial credentials are admin / changeme
  * To install additional capsule on separate machine continue by running:

      capsule-certs-generate --capsule-fqdn "$CAPSULE" --certs-tar "~/$CAPSULE-certs.tar"

  The full log is at /var/log/foreman-installer/katello.log
[ INFO 2016-08-02 11:19:00 verbose] All hooks in group post finished
```